src/iconsManifest/supportedFolders.ts

```diff
@@ -688,7 +688,13 @@ export const extensions: IFolderCollection = {
       format: FileFormat.svg,
       disabled: true,
     },
-    { icon: 'webpack', extensions: ['webpack'], format: FileFormat.svg },
+    { 
+      icon: 'webpack', 
+      extensions: [
+        'webpack',
+        '.webpack'
```

Comment:
What's this `.webpack` extension? is it a filename? If it is, then this is wrong.

Reply:
I'm going to remove it to ship the changes to tsconfig. Let's do another PR for this, if needed.